pkg/client/events_query/client_integration_test.go

```diff
@@ -0,0 +1,62 @@
+//go:build integration
```

Comment:
:disappointed: Mistakes were made; this file wasn't meant to be committed until de4defec5dd85d0182ba9cc1457bbc10f16a1e32. There will be modifications in that commit.